diff --git a/extensions/osx.bzl b/extensions/osx.bzl
index 1e25dc0..8f3e2f9 100644
--- a/extensions/osx.bzl
+++ b/extensions/osx.bzl
@@ -41,6 +41,7 @@ def _osx_extension_impl(mctx):
 
     for framework in frameworks:
         includes.append("System/Library/Frameworks/%s.framework/*" % framework)
+        includes.append("System/Library/PrivateFrameworks/%s.framework/*" % framework)
 
     # The following directories are unused, deprecated, or private headers.
     # These components:
